src/utils/types.ts

```diff
@@ -371,11 +372,12 @@ interface ResponseFull {
   list?: any // Used in ListBuilder
   date?: any // Used in DatePicker
   number?: number | null // Used in Number plugin
-  // Next 4 used in Checkbox Summary view
+  // Next 5 used in Checkbox Summary view
   textUnselected?: string
   textMarkdownList?: string
   textUnselectedMarkdownList?: string
   textMarkdownPropertyList?: string
+  values: { [key: string]: Checkbox }
```

Comment:
I don't really want to import Plugin types here, but there was no way without adding an ugly `any` hack to avoid errors.

Ideally, we should refactor this so that `ResponseFull` only has common properties, and we create separate types within each plugin that extend `ResponseFull`.